docs/reference/child_admin.rst

```diff
@@ -17,6 +21,7 @@ To do this, you first need to call the ``addChild`` method in your ``PlaylistAdm
 
             <call method="addChild">
                 <argument type="service" id="sonata.admin.video" />
+                <argument>playlist</<argument>
             </call>
         </service>
 
```

Comment:
below this, there is a section explaining the parentAssociation thing, should not be needed with this PR, please remove it.